tfhub_dev/assets/rishit-dagli/chest-xray-covid/1.md

```diff
@@ -0,0 +1,9 @@
+# Placeholder rishit-dagli/chest-xray-covid/1
+Lightweight deep learning model for clasifying chest xrays as COVID or normal.
+
+<!-- module-type: image-classification -->
+<!-- network-architecture: other -->
+<!-- dataset: other -->
+<!-- fine-tunable: false -->
+<!-- language: en -->
```

Comment:
The language metadata field is usually used for text models, for example to specify the language of the dataset the model was trained on. Since this is an image model we can remove this field - let me know if I'm missing anything.

Reply:
Thanks for pointing that out, I have now removed the field